tools/json2bin/consts/pokemon/evo_methods.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum, Flag, auto
+
+
+class EvoMethod(Flag):
```

Comment:
This needs to derive from `Enum`, not from `Flag`.

```suggestion
class EvoMethod(Enum):
```